packages/common-types/src/base/Polynomial.ts

```diff
@@ -69,7 +69,7 @@ class Polynomial implements ISerializable {
   getPublicPolynomial(): PublicPolynomial {
     const polynomialCommitments: Point[] = [];
     for (let i = 0; i < this.polynomial.length; i += 1) {
-      polynomialCommitments.push(getPubKeyPoint(this.polynomial[i]));
+      polynomialCommitments.push(getPubKeyPoint(this.polynomial[i], "secp256k1"));
```

Comment:
what about ed25519

Reply:
done

Reply:
Public Polynomial is is using `ecCurve` which is secp256k1

Please fix so that Public Polynomial computation to use `new EC("ed25519")`